js/screenSnippet/index.js

```diff
@@ -74,9 +87,17 @@ class ScreenSnippet {
             child = childProcess.execFile(captureUtil, captureUtilArgs, (error) => {
                 // will be called when child process exits.
                 if (error && error.killed) {
+                    // Method to reset always on top feature
+                    if (isAlwaysOnTop) {
+                        eventEmitter.emit('isAlwaysOnTop', true);
+                    }
                     // processs was killed, just resolve with no data.
                     resolve();
                 } else {
+                    // Method to reset always on top feature
+                    if (isAlwaysOnTop) {
```

Comment:
Can we eliminate duplicate code for the if condition and move it after the `if-else` block?